src/main/java/seedu/address/logic/commands/ListCommand.java

```diff
@@ -14,12 +14,12 @@
 
     public static final String COMMAND_WORD = "list";
 
-    public static final String MESSAGE_SUCCESS = "Listed all persons";
+    public static final String MESSAGE_SUCCESS = "%1$d students listed!";
```

Comment:
Maybe can add "all" at the front?
```suggestion
    public static final String MESSAGE_SUCCESS = "All %1$d students are listed!";
```

Reply:
True. Maybe we can change the verb? 
Otherwise, I think what you have now is fine

```suggestion
    public static final String MESSAGE_SUCCESS = "Displayed list with %1$d student(s)!";
```